scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -225,30 +226,30 @@ Don't skip this exploration!
 Bear in mind that the exact wrangling shown here will not be the same for other SCE objects you work with, but the same general principles apply.
 
 
-#### Preserving library information at the cell level
+#### Preserving sample information at the cell level
 
-How will we be able to tell which library a given cell came from?
+How will we be able to tell which sample a given cell came from?
 
-We must ensure that the column names of the SCE object are unique and directly add information to the SCE that indicates the different libraries.
-The best way to do this is simply to add a `colData` column with the library information.
+We must ensure that the column names of the SCE object are unique and directly add information to the SCE that indicates the different samples.
+The best way to do this is simply to add a `colData` column with the sample information.
```

Comment:
I flipped the order here so we can add the explanation about unique column names here, rather than in the bullet points above. So I copied some text from earlier (but have not directly suggested deleting it there; consider that suggestion implicit).

Update: There is a bit of repetition now with the next paragraphs, I realize, so this will still need a bit more revision! But I do think we say the same thing a few too many times as it stands!

```suggestion
The best way to do this is simply to add a `colData` column with the sample information.

We also must ensure that the column names of the SCE object are unique, even when combined with other SCE objects.
The current column names are cell ids (barcodes) that are only guaranteed to be unique _within_ a sample but may be repeated across samples.
This would be a problem for two reasons: 
First we wouldn't be able to identify the cell's originating sample.
Second, it would literally cause an error in R, which does not allow duplicate column names.
```